GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -632,21 +632,20 @@ private void HighlightBranch(ObjectId id)
 
         public string DescribeRevision(GitRevision revision, int maxLength = 0)
         {
+            var description = revision.IsArtificial
+                ? string.Empty
+                : "@" + revision.ObjectId.ToShortString(8) + ": ";
```

Comment:
I'd avoid the magic number
```suggestion
                : "@" + revision.ObjectId.ToShortString() + ": ";
```
and adapt the default argument of `ToShortString` instead.